omegatrade-app/backend/app/middlewares/jwt-auth.middleware.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = {
+    validateToken: (req, res, next) => {
+        const authorizationHeaader = req.headers.authorization;
+        if (authorizationHeaader) {
+            const token = req.headers.authorization.split(' ')[1]; // Bearer <token>
```

Comment:
Would it make sense to check that the result of the `split` function has a length of at least 2? An invalid header could in this case cause an unexpected error.

Reply:
This issue has been updated.